### DISCUSSION TITLE
amazing extension

### DISCUSSION THREAD

**Discussion by ride5k** [richrobber2/canvas-zoom#43] [General]
just wanted to say 1000x thanks, this makes masking WAAAAAY better!

**Comment by daswer123** [comment]
Bro, I'm glad you like it. I used to suffer in Inpaint too, but when I saw the basic functionality from @richrobber2 , I decided to develop it.

And thanks to everyone who suggested improvements, thanks to @richrobber2 basic code, We got such a cool result.

**Comment by richrobber2** [comment]
and it all got maintained and improved by you you really did do a good job on this project even recommending the extension to the repo that gets used by "everybody" on automatic
